site/docs/ch/flink/flink-get-started.md

```diff
@@ -74,6 +74,7 @@ cp ../flink-shaded-hadoop-2-uber-${HADOOP_VERSION}-10.0.jar lib
 
 ## Hive兼容
 Arctic 0.3.1 版本开始支持 Hive 兼容的功能，可以通过 Flink 读取/写入 Arctic Hive 兼容表数据。当通过 Flink 操作 Hive 兼容表时，需要注意以下几点：
+
 1. Flink Runtime Jar 不包括 Hive 依赖的 Jar 包内容，需要手动将[ Hive 依赖的 Jar 包](https://repo1.maven.org/maven2/org/apache/hive/hive-exec/2.1.1/hive-exec-2.1.1.jar)放到 flink/lib 目录下；
```

Comment:
Arctic DO NOT dependent on `hive-exec`, why should Flink need this dependency?

Reply:
The flink-runtime module has excluded the arctic-hive/hive-metastore dependency and shaded the parquet and avro dependency. So flink-runtime is running with hive-exec succeed.